### DISCUSSION TITLE
the new images are not yet tagged with v0.5.0

### DISCUSSION THREAD

**Discussion by manuelwallrapp** [tektoncd/results#399] [Q&A]
Hi there

I just saw, that the newly released images are not yet tagged with v0.5.0. Can anybody do that?

Best regards.. Manu

**Comment by khrm** [comment]
Were we not tagging images earlier? I would modify the release script. I would manually tag the images.

**Reply by vdemeester** [reply]
https://github.com/tektoncd/results/blob/main/release/release.sh#L17 Seems like they should be tagged though 🤔

**Reply by manuelwallrapp** [reply]
I don't see any v0.5.0 tags here: 
https://console.cloud.google.com/gcr/images/tekton-releases/global/github.com/tektoncd/results/cmd/api

**Comment by adambkaplan** [comment]
@vdemeester @khrm this still seems to be an issue - it appears our release pipeline used the commit SHA `9f84a1f` as the version - it shows up in the tag as well as the kubernetes application labels.

cc @enarha since you are working on the next release - #434

**Reply by khrm** [reply]
I would raise a pr tomorrow. @enarha ping me if I  get busy with something else.
